service_mesh/v2x/upgrading-ossm.adoc

```diff
@@ -18,6 +18,30 @@ To upgrade {ProductName}, you must create an instance of {ProductName} 2.0 in a
 
 .Procedure
 
+. Check your {ProductName} 1.1 `ServiceMeshControlPlane` resource configuration to make sure it's valid. If it's invalid, you must fix any invalid v1.x  configuration items before you continue. You can replace, patch or edit the resource with the `oc` command.
```

Comment:
v1.x is not correct.  The ServiceMeshControlPlane reource is invalid.  To fix it, it the v1 version of the resource needs to be modified.  Because v1 is no longer the default version for the resource, users must explicitly specify the version when retrieving, modifying, or patching the resource.  They cannot fix the resource with a v2 version, as the conversion to v2 fails.

From my previous comment, there are three ways to fix the invalid configuration: 1) retrieve, modify, replace; 2) patch; 3) edit.

Reply:
Also, we should probably include some text that tells the user how to identify the problem.  They can retrieve the v2 version (e.g. oc get smcp -o yaml) and look at the `spec.techPreview.errored.message` field.